### ISSUE TITLE
Error deploying kubeapps with Helm v2.12

### ISSUE THREAD

**Issue by drzero42** [kubeapps/kubeapps#888]
```
$ minikube version
minikube version: v0.31.0

$ minikube start
Starting local Kubernetes v1.10.0 cluster...

$ helm init
$HELM_HOME has been configured at /home/drzero/.helm.

$ helm repo add bitnami https://charts.bitnami.com/bitnami
"bitnami" has been added to your repositories

$ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps --set rbac.create=false
Error: release kubeapps failed: customresourcedefinitions.apiextensions.k8s.io "apprepositories.kubeapps.com" already exists
```

**Comment by andresmgot** [comment]
hi @drzero42,

I am not able to reproduce your issue with a fresh minikube cluster (v0.31.0) maybe there is some elements from a previous Kubeapps. If that is a testing environment you may be able to delete the instance (`minikube delete`) and recreate it again?

**Comment by drzero42** [comment]
I did this multiple times and deleted the minikube cluster in between, with the same result every time. I have even just tried it on a different machine and I can produce the error reliably every time.
Extra info that might help our triaging:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T21:04:45Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

$ helm version
Client: &version.Version{SemVer:"v2.12.0", GitCommit:"d325d2a9c179b33af1a024cdb5a4472b6288016a", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.12.0", GitCommit:"d325d2a9c179b33af1a024cdb5a4472b6288016a", GitTreeState:"clean"}

$ helm inspect bitnami/kubeapps | grep -i ^version
version: 1.1.0
```
Anything other info I can provide to help?

**Comment by prydonius** [comment]
Hmm, this actually looks like the same issue @davidkarlsen had. See Slack thread: 
https://kubernetes.slack.com/archives/C9D3TSUG4/p1545054473041700

**Comment by andresmgot** [comment]
You are right @drzero42, thanks for the debug information. Apparently it's an issue with helm v2.12 (released this month). We will investigate it. In the meantime, if you want to use Kubeapps you can downgrade helm to v2.11.

**Comment by andresmgot** [comment]
So this is an issue in Helm: https://github.com/helm/helm/issues/5054

It seems that they will cut a patch release soon. I think we could wait for that release. @prydonius do you think we should exclude the version 2.12.0 in the `tillerVersion` field of the `Chart.yaml`?

**Comment by prydonius** [comment]
@andresmgot I don't think it's worth it, the range for that will be difficult to do and honestly the error message from Helm isn't that helpful anyway. I imagine most people who google this will end up here, and hopefully more people will be aware once this is fixed in Helm.

@davidkarlsen you are also using helm v2.12.0 right?

**Comment by davidkarlsen** [comment]
@prydonius Yes - I was using 2.12.0 - downgrading to 2.11.0 fixed it.

**Comment by andresmgot** [comment]
great @bacongobbler, thanks for the update! I can confirm that the issue is fixed in 2.12.1. Closing this issue now!

**Comment by andresmgot** [comment]
BTW @drzero42, unrelated to this issue, but you don't need to disable RBAC (`rbac.create=false`). Minikube 0.31.0 enables RBAC by default.
